cli/src/commands/unsquash.rs

```diff
@@ -57,13 +58,13 @@ pub(crate) fn cmd_unsquash(
 ) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
     let commit = workspace_command.resolve_single_rev(&args.revision)?;
-    workspace_command.check_rewritable([&commit])?;
+    workspace_command.check_rewritable([commit.id()])?;
     let parents = commit.parents();
     if parents.len() != 1 {
         return Err(user_error("Cannot unsquash merge commits"));
     }
     let parent = &parents[0];
-    workspace_command.check_rewritable(&parents[..1])?;
+    workspace_command.check_rewritable(parents[..1].iter().ids())?;
```

Comment:
nit: `[parent.id()]`

Reply:
This was written in a strange way... I rewrote it so that the `parents` variable isn't created at all.